e2e/rsmgp_example_test/test_simple/test.yml

```diff
@@ -1,35 +0,0 @@
-query: >
```

Comment:
@jmatak can you rollback this as well + ignore the rust test in the e2e run?

Reply:
I should then include exclusion on e2e tests, which will introduce unnecessary complexity in other parts of the code (for example, testing action). I would leave it as clean as possible